src/pages/community/patterns/create-flows/index.mdx

```diff
@@ -3,367 +3,440 @@ title: Create flows
+   task. It can include message text and components.
+6. **Toggle (optional)**: Optional toggle allows the user to view all available
+   options in one scrolling page with anchor links.
+7. **Buttons**: The primary, secondary, and ghost buttons are 64px tall fluid
+   buttons. They are "Next", "Back" and "Cancel", respectively. The "Back"
+   button will be disabled on the first step. The "Next" button will change to
+   "Create" on the last step of creation. "Cancel" takes the user to the
+   previous page and cancels the create action.
+
+### Alignment
+
 <Row>
 <Column colLg={8}>
 
-![Show all options on](images/19.png)
-
-<Caption>Show all options on</Caption>
+![Full page alignment](images/FullPageAlignment.jpg)
 
 </Column>
 </Row>
 
+<InlineNotification>
+Coming soon from Carbon, 80px tall fluid buttons will be the new standard
+replacing the 64px fluid buttons in the bottom sticky footer. Image reference
+below.
+</InlineNotification>
+
 <Row>
 <Column colLg={8}>
 
-![Show all options on and grouped](images/20.png)
-
-<Caption>Show all options on and grouped</Caption>
+![Upcoming 80px fluid button height in the Full-page flow](images/FullPageAlignment80px.jpg)
 
+<Caption>Upcoming 80px fluid button height in the Full-page flow</Caption>
 </Column>
 </Row>
 
-### Confirmation
+### Behaviours, structure, and functionality
 
-A notification banner can be presented to visually confirm when any create
-action is a success.
+Full page Create flows can include an optional Toggle to show all the steps on a
+single scrolling page with anchor links. Toggling to "Show all available
+options" will replace the progress indicator with the In-page navigation anchor
+links.
 
-<Row>
-<Column colLg={8}>
-
-![Example of a successful creation confirmation](images/27.png)
+#### Do:
 
-<Caption>Example of a successful creation confirmation</Caption>
+- In Cloud and Cognitive, the Full-page create flow is the process the user 
+  must complete in order to use a product or service.
+- In IBM Cloud public, it is the alternative to the Provisioning flow for a
+  non-billable service.
 
-</Column>
-</Row>
+#### Don't:
 
-## Pre-create
+- Don't use the Full-page option when the product or service is usable before 
+  the current creation flow is initiated.
+- For IBM Public Cloud, do not use this when provisioning a service directly
+  from the Catalog, or when creating a non-billable resource that could happen
+  in a side panel/in context.
 
-A pre-create step occurs before the create flow starts. A common pre-create step
-is **Create from template**.
+## Creation
 
-A pre-create step is usually combined with a
-[full-page create flow](#full-page). It can also be used in combination with a
-[side panel create flow](#full-page). A pre-create step can be used in
-combination with a single-step or multistep create flow. A multistep flow starts
-after the pre-create step is complete.
+### Behaviors, structure and functionality
```

Comment:
These are supposed to be 3 distinct `h3` sections. I would start with calling this section just "Behaviors." I would then add some `h4` headings to each paragraph, suggestions below.

Reply:
done :)